lib/slack/real_time/concurrency/async.rb

```diff
@@ -43,11 +43,6 @@ def restart_async(client, new_url)
             end
           end
 
-          def disconnect!
-            super
-            @reactor.cancel
```

Comment:
@ioquatix Do we not need this anymore? Are we leaving the reactor running when we explicitly call `disconnect!` now?

Reply:
Let me check...